wagtailio/settings/base.py

```diff
@@ -264,12 +264,36 @@
 # Cache
 # Use Redis or database as the cache backend
 
-if "REDIS_URL" in env:
+# Prefer the TLS connection URL over non
+REDIS_URL = env.get("REDIS_TLS_URL", env.get("REDIS_URL"))
+
+if REDIS_URL:
+    connection_pool_kwargs = {}
+
+    if REDIS_URL.startswith("rediss"):
+        # Heroku Redis uses self-signed certificates for secure redis conections.
+        # https://stackoverflow.com/a/66286068
+        # When using TLS, we need to disable certificate validation checks.
+        connection_pool_kwargs["ssl_cert_reqs"] = None
+
+    redis_options = {
+        "IGNORE_EXCEPTIONS": True,
```

Comment:
Because of this, we should add `DJANGO_REDIS_LOG_IGNORED_EXCEPTIONS = True`

https://git.torchbox.com/internal/wagtail-kit/-/blob/master/wagtailkit_repo_name/wagtailkit_repo_name/settings/base.py#L200

Reply:
huh, that got lost somehow. re-added. thanks